src/fairchem/core/models/equiformer_v2/layer_norm.py

```diff
@@ -72,50 +72,51 @@ def __init__(
     def __repr__(self) -> str:
         return f"{self.__class__.__name__}(lmax={self.lmax}, num_channels={self.num_channels}, eps={self.eps})"
 
-    @torch.cuda.amp.autocast(enabled=False)
     def forward(self, node_input):
         """
         Assume input is of shape [N, sphere_basis, C]
         """
+        with torch.autocast(device_type=node_input.device.type, enabled=False):
```

Comment:
style nit, maybe use two decorators on the method (I think that should work just fine). I find it more explicit and less room to get an indentation wrong in the body of the method:
```python

@torch.autocast(device_type="cuda", enabled=False)
@torch.autocast(device_type="cpu", enabled=False)
def forward(self, node_input):
```